public-ui/src/app/views/quick-search/quick-search.component.html

```diff
@@ -35,15 +35,11 @@ <h5 class="box-title">
               {{r.name}}
             </h5>
             <div class="card-stat">
-              {{r.standardConceptCount | number}}
+              {{r.allConceptCount | number}}
             </div>
             <p class="card-text bigger">
-              <em> <b>concepts </b></em>
-              <span *ngIf="searchText.value"> match your search. </span>
-              <span *ngIf="!searchText.value">in this domain.</span>
-              <span *ngIf="r.participantCount">
-                <span class="bolder"> {{r.participantCount | number}}</span> participants in this domain
-              </span>
+              <span *ngIf="searchText.value; else no_search_value">matching medical concepts</span>
+              <ng-template #no_search_value><span>medical concepts</span></ng-template>
```

Comment:
I don't recall the big changes to this part being required. But if they are great.